servicex/expandable_progress.py

```diff
@@ -183,3 +184,16 @@ def advance(self, task_id, task_type):
                 self.progress.advance(task_id=self.overall_progress_download_task)
         elif self.display_progress and not self.overall_progress:
             self.progress.advance(task_id=task_id)
+
+
+class TranformStatusProgress(Progress):
+    def get_renderables(self):
+        for task in self.tasks:
+            if task.fields.get("bar") == "failure":
+                self.columns = [
+                    TextColumn("[progress.description]{task.description}"),
+                    BarColumn(complete_style="rgb(255,0,0)"),
```

Comment:
Why is this here and different from `complete_style="rgb(114,156,31)` above?

Reply:
And this red color is for when the bar is progressing but some files have failed.